src/lava/lib/dl/netx/hdf5.py

```diff
@@ -31,20 +31,27 @@ class Network(AbstractProcess):
         first ``num_layers`` blocks in the description. The actual number of
         generated layers may be less than ``num_layers``. If it is None, all
         the layers are generated. Defaults to None.
-    has_graded_input : bool, optional
-        flag indicating the input spike type of input layer. Defaults to False.
+    input_message_bits : int, optional
+        number of message bits in input spike. Defaults to 0 meaning unary
+        spike.
```

Comment:
if you accept my suggestion on the other file about 0 vs. 1 bits, don't forget to update this file too